### PR TITLE
Wrapping FileTraceWriter in a composite trace writer that is passed to the function.

### LINKED ISSUE #59: C# TextWriter Log needs to be written to Streaming Logs
## Issue

When using .csx, we can bind a `TextWriter log` and make calls like `log.Verbose("Foobar")` and the streaming log does not write the records to the console.

## Expected behavior

It should write the logs to the streaming logs (i.e. "Verbose: Foobar"), as does the Node experience.

## Mitigation attempted

I attempted to add the following to host.json, but it did not address the issue:

```JSON
    "tracing": {
      "consoleLevel": "Verbose"
    }
```

### PR DESCRIPTION
**Pull request by fabiocav**
Resolves #59

### CODE REVIEW

**Comment by fabiocav** [comment]
@mathewc this is the scenario I was describing.
Eventually, it would be good to use the CompositeTraceWriter in the WebJobs SDK so we don't have to maintain a new one.

File: src/WebJobs.Script/CompositeTraceWriter.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.Azure.WebJobs.Host;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    internal sealed class CompositeTraceWriter : TraceWriter
```
Comment (mathewc) [inline src/WebJobs.Script/CompositeTraceWriter.cs]: Add a TODO comment on here that this is a dupe of fx class until it becomes public (if it does).

**Comment by mathewc** [comment]
Recommend updating one of your e2e tests so that it writes to a TraceWriter and you verify the output log file contains your log entry. You'll have to turn on file logging on the ScriptHostConfiguration - there's a property for it.

File: src/WebJobs.Script/Description/CSharp/CSharpFunctionInvoker.cs [hunk 2]
```diff
@@ -170,6 +172,20 @@ private void RestorePackages()
             }
         }
 
+        private object[] ProcessInputParameters(object[] parameters)
+        {
+            for (int i = 0; i < parameters.Length; i++)
+            {
+                TraceWriter writer = parameters[i] as TraceWriter;
```
Comment (mathewc) [inline src/WebJobs.Script/Description/CSharp/CSharpFunctionInvoker.cs]: Note that the SDK also supports TextWriter parameters. TraceWriter is relatively new (added by me). I'm fine with saying that we only support log stream support for TraceWriter for now, but just a heads up if you see people try TextWriter. SDK users will try that.

File: src/WebJobs.Script/CompositeTraceWriter.cs (hunk 1 above)
Comment (fabiocav) [inline src/WebJobs.Script/CompositeTraceWriter.cs]: :+1:

File: src/WebJobs.Script/Description/CSharp/CSharpFunctionInvoker.cs (hunk 2 above)
Comment (fabiocav) [inline src/WebJobs.Script/Description/CSharp/CSharpFunctionInvoker.cs]: I'm in favor of keeping it as is and revisiting if needed.

**Pull request merged**